_data/staff.yml

```diff
@@ -120,6 +120,12 @@
   image: "c-turner"
   email: "chris@axiomdatascience.com"
 
+- name: "Iwen Su"
```

Comment:
Your bio should go between Brian Stone and Dr. Kristen Thyng (staff are mostly listed alphabetically, after the founders). Otherwise looks great! I thought the double quotes in your description might cause an issue but I see that they're not the standard double quote character so should be all good, clever.